app/jobs/bulkrax/create_relationships_job.rb

```diff
@@ -41,7 +41,7 @@ def perform(parent_identifier:, importer_run_id:)
       @child_records = { works: [], collections: [] }
       pending_relationships.each do |rel|
         raise ::StandardError, %("#{rel}" needs either a child or a parent to create a relationship) if rel.child_id.nil? || rel.parent_id.nil?
-        _, child_record = find_record(rel.child_id, importer_run_id)
+        @child_entry, child_record = find_record(rel.child_id, importer_run_id)
```

Comment:
does this need to be an instance variable?

Reply:
mmm, nope!

Reply:
yes. otherwise `child_entry` in the rescue below will be nil.